staff/albert-ferrer/precourse/pasapalabra/Pasapalabra.js

```diff
@@ -0,0 +1,178 @@
+  { letter: "z", answer: "zen", status: 0, question: "CON LA Z. Escuela de budismo que busca la experiencia de la sabiduría más allá del discurso racional"},
+]
+
+
+
+
+//Function wellcomeUser para dar la bienvenida al usuario.
+const wellcomeUser=()=>{
+  
+  alert(`Estamos encantados de tenerte con nosotros, que tengas muchas suerte ${usuario}.`);
+
+}
+
+//Function "preguntas" donde se hace la primera ronda de preguntas. Las preguntas sin respuesta o errones contaran como error. Las preguntas que usen "pasapalabra irán a la siguiente ronda."
+const preguntas=() =>{
+
+  for(const index in questions){
+
+      let preg = prompt(questions[index].question);
+
+    if(preg === questions[index].answer){
+      alert(`¡Respuesta correcta! Sigamos.`);
+      questions[index].status = 'yes';
+      respCorrecta +=1;
+
+    } else if(preg === 'pasapalabra'){
+      alert(`¡Dejamos la pregunta para la siguiente ronda!Siguiente pregunta.`);
+      questions[index].status = 'next';
+        
+
+    } else if(preg != questions[index].answer || preg === ''){
+      alert(`¡Respuesta equivocada!Vamos a por la siguiente.`);
+      questions[index].status = 'no';
+      respIncorrecta +=1;
+
+    }
+  }
+}
+
+//Function "siguienteRonda" que devolverá las preguntas que han sido respuestas con "pasapalabra".
+const siguienteRonda=()=>{
+
+   for(const index in questions){
+
+     if(questions[index].status === 'next'){
+
+       let preg = prompt(questions[index].question);
+    
+       if(preg === questions[index].answer){
+        alert(`¡Respuesta correcta! Sigamos.`);
+        questions[index].status = 'yes';
+        respCorrecta+=1;
+
+     } else if(preg === 'pasapalabra'){
+      alert(`¡Dejamos la pregunta para la siguiente ronda!Siguiente pregunta.`);
+      questions[index].status = 'next';
+
+     } else if(preg != questions[index].answer || preg === ''){
+      alert(`¡Respuesta equivocada! Vamos a por la siguiente.`);
+      questions[index].status = 'no';
+      respIncorrecta+=1; 
+     
+     }
+   }
+ } 
+   
+}
+
+ //Function "statusRonda" que checkea si todas las respuestas han sido respondidas(tanto correctamente como erroneas). Si no es así volverá a ejecutar la function anterior.
+const statusRondas=()=>{
+  let sumaResp = respIncorrecta + respCorrecta;
+
+  if(sumaResp != 27);
+  {
+    siguienteRonda();
+    
+  }
+  
+}
+
+//Muestra de la clasificación, con el nombre del usuario + las respuestas acertadas.
+/*const userRanking=()=>{
+
+  clasificacion.push({nombre: usuario, pregAcertadas: respCorrecta});
+  clasificacion.sort(function(a,b){
+     return (b.respCorrecta - a.respCorrecta);
+ });
+
+  for(const posicion in clasificacion){
+
+    alert(`Skylaber: ${clasificacion[posicion].nombre}, con un total de preguntas acertadas: ${clasificacion[posicion].pregAcertadas}`);
+
+  }
+ puntuacion();
+}
+*/
+function useRanking(){
```

Comment:
Pay attention here, you wrote "useRanking" (instead of "userRanking") so when you call the function at the end writing "userRanking" the function is returning an error.